Gemfile

```diff
@@ -0,0 +1,16 @@
+source "https://rubygems.org"
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+# Specify your gem's dependencies in activejob-web.gemspec.
+gemspec
+
+gem "puma"
+
+gem 'pg'
+
+gem "sqlite3"
```

Comment:
@MohammedNazeer10 As we decided to gow with pg, So why we are still having this sqlite3 gem in Gemfile?

Reply:
@rajeshkanna-mallow , i have removed and updated PR